db/repositories/run_repository.py

```
@@ -0,0 +1,75 @@
+        run = Run(
+            run_id=run_id,
+            created_at=ts,
+            total_turns=config.num_turns,
+            total_agents=config.num_agents,
+            started_at=ts,
+            status=RunStatus.RUNNING,
+        )
+        try:
+            write_run(run)
+        except Exception as e:
+            raise RuntimeError(f"Failed to create run {run_id}: {e}") from e
```

Ruff (0.14.6)

Avoid specifying long messages outside the exception class

(TRY003)